### PR TITLE
Sprint #3 Merge Request

### PR DESCRIPTION
**Pull request by Jyyunrin**

### CODE REVIEW

**Comment by R-Lok** [comment]
LGTM.

File: public/scripts/incrementStreak.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
+// require('dotenv').config({ path: `${process.cwd()}/.env.${process.env.NODE_ENV}` })
+const usersModel = require('../../models/users')
```
Comment (wyliao76) [inline public/scripts/incrementStreak.js:2]: This script is weird.

File: views/generate.ejs [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+              </div>
+            </div>
+          </form>
+        </div>
+
+        <div class="tab-pane fade" id="photo-tab-pane" role="tabpanel" aria-labelledby="photo-tab" tabindex="0">...
+        </div>
+      </div>
+    </div>
+<<<<<<< HEAD
```
Comment (wyliao76) [inline views/generate.ejs:86]: remove

File: views/generate.ejs [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+            <button type="button" id="generatePhotoButton" class="btn btn-secondary">Generate</button>
+            <!-- <button type="button" class="btn btn-primary"></button> -->
+          </div>
+        </div>
+      </div>
+    </div>
+    <script type="text/javascript" src="../scripts/generate.js"></script>
+=======
+    <script src="scripts/resetPassword.js"></script>
+>>>>>>> jimmy_kong_signup-login
```
Comment (wyliao76) [inline views/generate.ejs:114]: remove.

File: views/settings.ejs [hunk 4]
```diff
@@ -0,0 +1,201 @@
...
+      <div class="modal fade" id="profileModal" tabindex="-1" role="dialog" aria-labelledby="exampleModalCenterTitle" aria-hidden="true">
+        <div class="modal-dialog modal-dialog-centered" role="document">
+          <div class="modal-content">
+            <div class="modal-header justify-content-center">
+              <h5 class="modal-title" id="profileModalTitle">Choose your Profile Picture</h5>
+            </div>
+            <div class="modal-body">
+              <div class="container-fluid">
+                <div class="row">
+                  <% for (let i = 1; i <= 9; i++) { %>
```
Comment (wyliao76) [inline views/settings.ejs:29]: smells

**Review by wyliao76 — APPROVED** [review]
It works.
We will refactor and cleanup next sprint.

**Pull request merged**